operational/aws/long_running_instances/long_running_instances.pt

```diff
@@ -0,0 +1,332 @@
+    encoding "xml"
+    collect xpath(response, "//DescribeInstancesResponse/reservationSet/item", "array") do
+      field "instances_set" do
+        collect xpath(col_item,"instancesSet/item","array") do
+          field "region",val(iter_item, "region")
+          field "instanceId", xpath(col_item,"instanceId")
+          field "imageId", xpath(col_item,"imageId")
+          field "instanceType", xpath(col_item, "instanceType")
+          field "platform", xpath(col_item, "platform")
+          field "privateDnsName", xpath(col_item, "privateDnsName")
+          field "launchTime", xpath(col_item, "launchTime")
+          field "tag_set" do
+            collect xpath(col_item,"tagSet/item","array") do
+              field "key", xpath(col_item, "key")
+              field "value", xpath(col_item, "value")
+            end
+          end
+        end
+      end
+    end
+  end
+end
+
+# get AWS instance size map
+datasource "ds_aws_instance_size_map" do
+  request do
+    verb "GET"
+    host "raw.githubusercontent.com"
+    path "/rightscale/policy_templates/master/data/aws/instance_types.json"
+    header "User-Agent", "RS Policies"
+  end
+end
+
+###############################################################################
+# Scripts
+###############################################################################
+
+script "js_instances", type: "javascript" do
+  result "results"
+  parameters "ds_instance_set", "param_exclusion_tag_key"
+  code <<-EOS
+  var tag_key = param_exclusion_tag_key.split(':')[0]
+  var tag_value = param_exclusion_tag_key.split(':')[1]
+  var results = []
+  for ( n=0; n < ds_instance_set.length; n++) {
+    var instance_set = ds_instance_set[n].instances_set
+    for ( i=0; i < instance_set.length; i++) {
+      var instance = instance_set[i]
+      var tags = instance.tag_set
+      if (_.contains(_.pluck(tags,'key'), tag_key) && _.contains(_.pluck(tags,'value'), tag_value)){
+        continue
+      } else {
+        var now = new Date()
+        var one_day=1000*60*60*24
+        var launchTime = new Date(instance.launchTime)
+        var difference = now.getTime() - launchTime.getTime()
+        var days_old=(difference/one_day).toFixed(2)
+        instance["hostname"] = instance.privateDnsName.split('.')[0]
+        instance["days_old"] = days_old
+        results.push(instance)
+      }
+    }
+  }
+
+  results = _.sortBy(results, 'region');
+EOS
+end
+
+###############################################################################
+# Policy
+###############################################################################
+
+policy "pol_utilization" do
+  validate_each $ds_instances do
+    summary_template "{{ rs_project_name }} (Account ID: {{ rs_project_id }}): {{ len data }} AWS instances running longer than {{ parameters.param_days_old}} days"
+    detail_template <<-EOS
+# AWS Long Running Instances
+| Region | instanceID | instanceType | Hostname | Private DNS Name | Days Old | Tags |
```

Comment:
instanceType should be Instance Type.  consider making the second column and sorting by region then instance type.